outputs.tf

```diff
@@ -0,0 +1,24 @@
+output "connection_string" {
+  value       = azurerm_eventhub_namespace_authorization_rule.this.primary_connection_string
+  description = "Event Hub namespace connection string"
+}
+
+output "name" {
```

Comment:
consider renaming the output to a namespace

Reply:
done